### PR TITLE
Feature/claudia_5427 Adding management of version in glancesync component

### PR DESCRIPTION
**Pull request by flopezag**
#### Reviewers

 @jframos, @pratid , @chemaper  
#### Description
- Resolution of version management in setup.py
- Configuration of bumpversion
#### Testing

 No test.

### CODE REVIEW

**Comment by chemaper** [comment]
LGTM

File: .bumpversion.cfg [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[bumpversion]
+current_version = 1.0.0
+commit = True
+tag = True
+
```
Comment (pratid) [inline .bumpversion.cfg:5]: We prefer not creating tags when bumping a new version, as this operation is performed at the very beginning of a new development sprint, just after previous version is tagged and released from master branch. So please set `tag` to False.

File: .bumpversion.cfg [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+[bumpversion:file:setup.py]
+search = __version__ = {current_version}
+replace = __version__ = {new_version}
```
Comment (pratid) [inline .bumpversion.cfg]: I guess the single quotes (`'`) are required here in order to match the text to replace

Comment (pratid) [inline .bumpversion.cfg]: Don't forget to include additional files in which the version number is included:
- `sonar-project.properties`

**Comment by pratid** [comment]
LGTM

**Comment by jframos** [comment]
LGTM

**Pull request merged**